user_scripts/fielded_batch_ingest.py

```diff
@@ -324,25 +324,25 @@ def verifyURL(url):
     }    
     try:
         req = HeadRequest(url, None, headers)
-        u = urllib2.urlopen(req)
-    except ValueError, e:
-        print str(e) + '; skipping invalid URL ' + url
+        u = urllib.request.urlopen(req)
+    except ValueError as e:
+        print(str(e) + '; skipping invalid URL ' + url)
         return False
-    except urllib2.HTTPError, e:
+    except urllib.error.HTTPError as e:
         if e.code in (405, 501):
             # Try a GET request (HEAD refused)
             # See also: http://www.w3.org/Protocols/rfc2616/rfc2616.html
             try:
-                req = urllib2.Request(url, None, headers)
-                u = urllib2.urlopen(req)
+                req = urllib.request.Request(url, None, headers)
+                u = urllib.request.urlopen(req)
             except:
-                print str(e) + '; Response: ' + str(e.code) + '; skipping URL ' + url
```

Comment:
I believe we still need to do `str` of e and e.code because they are not strings. Same below.